pkg/cosign/verify.go

```diff
@@ -634,13 +639,18 @@ func verifySignatures(ctx context.Context, sigs oci.Signatures, h v1.Hash, co *C
 		bundleVerified = bundleVerified || verified
 	}
 
-	if len(checkedSignatures) == 0 {
+	if len(checkedSignatures) == 0 && t.Err() != nil {
```

Comment:
I think we can remove `&& t.Err() != nil` from this check, if we have no valid signatures then we always have to return an error